testing/run_tests.py

```diff
@@ -389,7 +389,7 @@ def main():
   if 'benchmarks' in types and not IsWindows():
     RunEngineBenchmarks(build_dir, engine_filter)
 
-  if 'engine' in types or 'font-subset' in types:
+  if ('engine' in types or 'font-subset' in types) and args.variant != 'host_release':
```

Comment:
@dnfield : the font-subset doesn't seem to be available in release mode. I also wonder if it's available in the profile mode. Is it debug mode only?

Reply:
I think only the debug mode asset is built - it should be possible to build the release mode, but not important to test it in release mode.